app/views/posts/index.html.erb

```diff
@@ -1,21 +1,21 @@
+<div>
+  <%= render 'partials/user' %>
+</div>
```

Comment:
- [ ] Style is not important in this case, but as you can see comments looks not correct. Kindly fix it.

- [ ] Also, ı open the posting page with the writing route on the address bar. It will be good if you make all posts clickable so it can directly open the page. (OPTIONAL)

![image](https://user-images.githubusercontent.com/38635158/185517556-570870c8-e3a1-4334-8cec-d75e081bc1f7.png)